Aptoide/appcoins-wallet-android
# [APPC-1505] Improvements to top up

**What does this PR do?**

    1-Added No network screen 
    2-Remove landscape from top up 
    3-User is now able to do a top up even if there's no bonus 
    4-Refactored code to better handle errors

**Database changed?**

No

**Where should the reviewer start?**

TopUpFragment.kt

**How should this be manually tested?**

  Check the use case described in the ticket and several others related to what was done (mainly mocking errors) 

**What are the relevant tickets?**

  https://aptoide.atlassian.net/browse/APPC-1505

**Code Review Checklist**
